scripts/Dockerfile

```
@@ -0,0 +1,16 @@
+FROM debian:testing
+
+ENV RISCV_DIR=/toolchain/riscv64-unknown-elf-gcc-8.3.0-2020.04.1-x86_64-linux-ubuntu14/bin
+ENV PATH="/renode:${RISCV_DIR}:${PATH}"
+ENV WORKDIR=/CFU-Playground
```

Won't this only needed during the build time? In that time probably we can change it to `ARG`
```suggestion
ARG WORKDIR=/CFU-Playground
```